backend/Properties/launchSettings.json

```diff
@@ -1,8 +1,8 @@
-﻿{
+{
```

Comment:
This file is not changed, your IDE has been doing autoformatting. This is a bad thing

Reply:
Are you sure it is just auto formatting, line 24 seems to have been removed by the IDE?
And as an addendum a linter would have caught the unnecessary formatting?